daemon/top.go

```diff
@@ -68,5 +68,6 @@ func (daemon *Daemon) ContainerTop(name string, psArgs string) (*types.Container
 			}
 		}
 	}
+	container.LogEvent("top")
```

Comment:
I added this one because my thinking was that anytime we do **anything** to/with a container we need to log it so that people can debug stuff.  In this case we started another process in the container so someone watching the event log might need to know that info.